worlds/subnautica/docs/setup_en.md

```diff
@@ -18,20 +18,26 @@
 
 4. Start Subnautica. You should see a Connect Menu in the topleft of your main Menu.
 
-## Troubleshooting
+## Connecting
 
-If you don't see the connect window check that you see a qmodmanager_log-Subnautica.txt in Subnautica, if not
-QModManager4 is not correctly installed, otherwise open it and look
-for `[Info   :   BepInEx] Loading [Archipelago 1.0.0.0]`, version number doesn't matter. If it doesn't show this, then
-QModManager4 didn't find the Archipelago mod, so check your paths.
+Using the Connect Menu in Subnautica's Main Menu you enter your connection info to connect to an Archipelago Multiworld.
+Menu points:
+ - Host: the full url that you're trying to connect to, such as `archipelago.gg:38281`.
```

Comment:
```suggestion
 - Host: the full url and port that you're trying to connect to, such as `archipelago.gg:38281`.
```

Reply:
url includes port, so that change is not required. With the sample on the right people will understand anyway.